frontend/website/src/Developers.re

```diff
@@ -0,0 +1,160 @@
+let extraHeaders = () =>
+  <>
+    <script src="https://apis.google.com/js/api.js" />
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.7.0/marked.min.js"
+      integrity="sha256-0Ed5s/n37LIeAWApZmZUhY9icm932KvYkTVdJzUBiI4="
+      crossOrigin="anonymous"
+    />
+  </>;
+
+module Styles = {
+  open Css;
+  let page =
```

Comment:
instead of copying a bunch of these over from `Testnet.re`, could we consolidate the styles?